### ISSUE TITLE
1Password recognizes tag field as password

### ISSUE THREAD

**Issue by niklas2810** [firefly-iii/firefly-iii#3805]
**Bug description**
I am running Firefly III version 3.5.5, and my problem is:

When creating a new transaction, my password manager 1Password (I'm using the Firefox add-on) recognizes the `tag` field on the right hand side as a password field (screenshot down below). I'm not quite sure where to put this issue, in this repo or directly forward it to 1Password. If just wanted to let you know of this strange behavior, maybe you can do something about it. Otherwise, feel free to close this issue :) 

**Steps to reproduce**

1. Open your Firefly III instance or the demo site.
2. Create a new transaction.
3. Try to enter a tag.
4. If 1Password is locked, a pop-up will appear. If it was unlocked, an icon will appear at the end of the line.

(The tag field is the only one showing such strange behavior.)

**Extra info**

I'm using 1PasswordX for Firefox on a Windows 10 machine. Firefly version is 5.3.3, 1PasswordX version is 1.21.0

**Bonus points**

- [x] I searched and nobody reported this bug before
- [x] I have added a screenshot.
- [x] I was able to replicate it on the demo site https://demo.firefly-iii.org/

![image](https://user-images.githubusercontent.com/30748216/93611614-cf9d6780-f9ce-11ea-8f54-e820c9806e53.png)

**Comment by JC5** [comment]
Weird. By default this is the tags input HTML, which shouldn't trigger 1Password:

```html
<input data-v-61d92e31="" placeholder="Tags" type="text" size="1" title="Tags" classes="form-input" class="ti-new-tag-input ti-valid">
```

I would appreciate a ticket with 1Password, so I know what's happening.

**Comment by niklas2810** [comment]
With the latest version of 1Password (1.22.1), this issue has been resolved because the pop-ups won't show up anymore. There is nothing wrong with the page configuration/HTML, it's just the 1Password algorithm trying to guess if there is a password field inside this page, and this particular format seems to trigger something. As far as I can see, there is nothing which can be changed on Firefly's side.

Additionally, I've found this forum thread: https://discussions.agilebits.com/discussion/113698/how-to-stop-1password-x-suggesting-emails-in-a-form-field This is about quite a similar scenario, and the 1Password team suggests to simply disable 1Password auto-fill for the site if it doesn't work properly.
